Added gallery tab for eQSL cards

stevendodd opened the pull request:
Added gallery tab with automatically generated thumbnails onclick brings up the full card as per the view button from the list. Requires phpX.X-gd `sudo apt-get install php8.1-gd`.

New Tab Screenshot
---
<img width="1363" height="691" alt="New Tab Screenshot" src="https://github.com/user-attachments/assets/9499b592-831c-4ea5-901e-49300094e2f2" />

Gallery Tab Screenshot
---
<img width="1363" height="691" alt="Gallery Tab Screenshot" src="https://github.com/user-attachments/assets/1458460a-9ab2-4a7f-9496-a16ca9b2b5d8" />

Onclick popup Screenshot
---
<img width="1363" height="691" alt="Onclick popup Screenshot" src="https://github.com/user-attachments/assets/aa9eca57-c4ab-4287-a23f-a98e70d4ccb9" />

magicbug commented:
Thanks for the PR @stevendodd if you would consider some of the PR feedback both from myself and the automated copilot feedback

stevendodd commented on application/views/eqslcard/index.php:
```diff
@@ -55,5 +70,91 @@
+				 }
+				 echo '<a href="#" onclick="viewEqsl(\''.$qsl->image_file.'\', \''. $qsl->COL_CALL . '\')" class="photo" style="background-image: url(/'.$dest.');"></a>';
+		     }
+		 }
```
Each time the page is loaded it will only generate a maximum of 100 thumbnails.. the user will need to reload the page to generate the next potential 100

magicbug commented:
> > Thanks for the PR @stevendodd if you would consider some of the PR feedback both from myself and the automated copilot feedback
> 
> Not sure how to handle the scrips/css do you have any suggestions?

Usually they live in assets folder should just be able to place them under each, you will find in footer.php lots of JS scripts are called along with header.php its looking good

stevendodd commented:
> > > Thanks for the PR @stevendodd if you would consider some of the PR feedback both from myself and the automated copilot feedback
> > 
> > 
> > Not sure how to handle the scrips/css do you have any suggestions?
> 
> Usually they live in assets folder should just be able to place them under each, you will find in footer.php lots of JS scripts are called along with header.php its looking good

Yes, I found it and addressed all the issues. It should be good to merge.
